### PR TITLE
doc(troubleshooting): improve ,,no result'' section in troubleshooting guide

### PR DESCRIPTION
**Pull request by derSascha**
Add some more infos how to handle empty result (might be related to #930 and #931).

---
This change is [Reviewable](https://reviewable.io/reviews/spark-jobserver/spark-jobserver/934)

### CODE REVIEW

File: doc/troubleshooting.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ If your job returns a large job result, it may exceed Akka's maximum network mes
 
 ## Job with status finished has no result
 
-On jobs with large results or many concurrent jobs, the REST API at `/job/abc..` might return status `FINISHED` but does not contains any result. This might happen in two cases:
+On jobs with large results or many concurrent jobs, the REST API at `/job/abc..` might return status `FINISHED` but does not contains any result. This might happen in three cases:
```
Comment (noorul) [inline doc/troubleshooting.md]: s/contains/contain

File: doc/troubleshooting.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ If your job returns a large job result, it may exceed Akka's maximum network mes
...
 
-1. The job finished right now and results are in transfer.
-2. The job finished some time ago and results are remove from results cache already. See `spark.jobserver.job-result-cache-size` to increase the cache.
+1. The job finished right now and results are in transfer between the akka actors. Try it again some milliseconds later.
+2. The job finished some time ago and results are remove from results cache already. See `spark.jobserver.job-result-cache-size` to increase the cache size.
```
Comment (noorul) [inline doc/troubleshooting.md]: s/remove/removed

**Comment by derSascha** [comment]
@noorul typos fixed

**Comment by noorul** [comment]
Merged, thank you!

**Pull request merged**